ypathgen/pathgen_test.go

```diff
@@ -1140,6 +1140,97 @@ func TestGeneratePathCodeSplitFiles(t *testing.T) {
 	}
 }
 
+func TestGeneratePathCodeSplitModules(t *testing.T) {
+	tests := []struct {
+		name                      string   // Name is the identifier for the test.
+		inFiles                   []string // inFiles is the set of inputFiles for the test.
+		inIncludePaths            []string // inIncludePaths is the set of paths that should be searched for imports.
+		inTrimOCPath              bool
+		inListBuilderKeyThreshold uint
+		wantStructsCodeFiles      map[string]string // wantStructsCodeFileDir map from package name to want source file.
+	}{{
+		name:    "oc simple",
+		inFiles: []string{filepath.Join(datapath, "openconfig-simple.yang")},
+		wantStructsCodeFiles: map[string]string{
+			"openconfigsimple": "testdata/modules/oc-simple/simple.txt",
+			"device":           "testdata/modules/oc-simple/device.txt",
+		},
+	}, {
+		name:         "oc simple and trim",
+		inFiles:      []string{filepath.Join(datapath, "openconfig-simple.yang")},
```

Comment:
Do you want to test openconfig-import.yang in order to test multiple packages being written?

Reply:
added